revokefs/demo.c

```diff
@@ -0,0 +1,85 @@
+#include <glib.h>
+#include <stdio.h>
```

Comment:
```suggestion
#include <stdio.h>
#include <stdlib.h>
```

Without this it fails to build on Debian-like distros:

```
revokefs/demo.c: In function ‘main’:
revokefs/demo.c:27:7: warning: implicit declaration of function ‘exit’ [-Wimplicit-function-declaration]
       exit (EXIT_FAILURE);
       ^~~~
revokefs/demo.c:27:7: warning: incompatible implicit declaration of built-in function ‘exit’
revokefs/demo.c:27:7: note: include ‘<stdlib.h>’ or provide a declaration of ‘exit’
revokefs/demo.c:5:1:
+#include <stdlib.h>
 
revokefs/demo.c:27:7:
       exit (EXIT_FAILURE);
       ^~~~
revokefs/demo.c:27:13: error: ‘EXIT_FAILURE’ undeclared (first use in this function)
       exit (EXIT_FAILURE);
             ^~~~~~~~~~~~
```